kubernetes/resource_kubernetes_replication_controller_test.go

```diff
@@ -1247,3 +1253,15 @@ func testAccKubernetesReplicationControllerConfigMinimal(name, imageName string)
 }
 `, name, name, name, name, imageName)
 }
+
+func getReplicaCount() int {
+	envReplicas, ok := os.LookupEnv("TF_ACC_KUBERNETES_RC_REPLICAS")
+	if ok {
+		rv, err := strconv.Atoi(envReplicas)
+		if err == nil {
+			return rv
+		}
+		fmt.Printf("Failed to parse value of TF_ACC_KUBERNETES_RC_REPLICAS env var: %s", err)
+	}
+	return 500 // historical default value
```

Comment:
I honestly wouldn't bother preserving this old value – I think 500 was probably excessively high. The comment next to that count said it was to test the waiter – but we can easily test the waiter by setting higher thresholds for liveness and readiness probes. Let's maybe just default it down to `50`?